client/api/smb/v1beta2/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
```

Comment:
There's this rule in the repo that pb.go files that were already released shouldn't be modified, because smb already reached v1 I guess that we'd need to create a new v2alpha1 API version, cc @ddebroy

Reply:
Actually i am not able to generate respective pb.go file so when i used this version i am able to do so. That's why i have used this version? do i need to use 1.26v only?

Reply:
I think that using the latest version of protoc-gen-go works however my point is about the file that's getting modified, @ddebroy said (and it's also pointed out in [the docs](https://github.com/kubernetes-csi/csi-proxy/blob/master/docs/DEVELOPMENT.md#making-changes-to-the-client-module)) that once the api.pb.go file of a version is released then it shouldn't be modified, we already released smb/v1beta2 so I think it should stay as it is.

Instead we could:
- bump the SMB version from v1 to v2alpha1
- make the changes in the proto file like adding the csi secrets annotation
- generate `client/api/smb/v2alpha/api.pb.go`

I'm still not sure how we'd use the csi secrets annotation, to check it I would: 

- see where it's used in the CSI Spec (which methods and messages)
- read the CSI sidecar or the CSI Driver code that's using the method/message and see what it's doing with the field, is it checking that the field has a property to do special logging?